tests/src/JIT/HardwareIntrinsics/X86/Shared/AesUnOpTest.template

```diff
@@ -52,6 +52,12 @@ namespace JIT.HardwareIntrinsics.X86
                 // Validates passing a static member works
                 test.RunClsVarScenario();
 
+                if ({LoadIsa}.IsSupported)
```

Comment:
These are the additional scenarios that were added to other templates in https://github.com/dotnet/coreclr/pull/23629